core/src/util/mod.rs

```diff
@@ -6,10 +6,9 @@ use std::mem;
 use std::ops::{Mul, Rem, Shr};
 
 mod int128;
-mod spotify_id;
 
+pub use spotify_id::{FileId, SpotifyId};
```

Comment:
This is reexporting the two types as `core::FileId` and `core::SpotifyId`.
I don't think this line is needed at all.